tests/testthat/setup.R

```diff
@@ -1,4 +1,3 @@
 pre_test_options <- options(
-  readr.show_col_types = FALSE,
```

Comment:
I prefer to leave this unset and let `should_show_types()` fill in `FALSE` in the testing context.